CART/CARTLib/examples/MultiContrastSegmentationEvaluationTask.py

```diff
@@ -0,0 +1,483 @@
+import json
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import ctk
+import qt
+import slicer
+from slicer.i18n import tr as _
+import json
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import ctk
+import qt
+import slicer
+from slicer.i18n import tr as _
+from .MultiContrastSegmentationEvaluationDataUnit import MultiContrastSegmentationEvaluationDataUnit
+from ..core.TaskBaseClass import TaskBaseClass, DataUnitFactory
+from ..utils.widgets import CARTSegmentationEditorWidget
+from ..utils.data import save_segmentation_to_nifti
+from ..LayoutLogic import CaseIteratorLayoutLogic
+
+
+VERSION = 0.01
+
+# TODO Allow for the user to select "All" which will create a row instead of a single view
+class MultiContrastSegmentationEvaluationGUI:
+    def __init__(self, bound_task: 'MultiContrastSegmentationEvaluationTask'):
+        self.bound_task = bound_task
+        self.data_unit: Optional[MultiContrastSegmentationEvaluationDataUnit] = None
+
+        # Layout logic for creating linked slice views
+        self.layoutLogic = CaseIteratorLayoutLogic()
+        self.currentOrientation: str = 'Axial'
+
+        # Widgets we’ll need to reference later:
```

Comment:
[Extreme nit] You use `’` rather than `'` here; it is in a comment, but I know some old-hat devs whose file editors would have a stroke trying to read this.

```suggestion
        # Widgets we'll need to reference later:
```